packages/augur-ui/src/modules/events/actions/listen-to-updates.ts

```diff
@@ -62,6 +64,13 @@ const EVENTS = {
   [SubscriptionEventName.WarpSyncHashUpdated]: wrapLogHandler(
     handleWarpSyncHashUpdatedLog
   ),
+  [SubscriptionEventName.OrderEvent]: wrapLogHandler(
```

Comment:
UI listens for bulk zerox order events not individual order events there is too much chatter and causes too many getter calls. 

`[SubscriptionEventName.BulkOrderEvent]: wrapLogHandler(handleBulkOrdersLog),`

Need to follow the same pattern as bulk order events.

Reply:
Fixed!